app/controllers/insured/consumer_roles_controller.rb

```diff
@@ -292,7 +292,7 @@ def help_paying_coverage
       save_faa_bookmark(request.original_url)
       set_admin_bookmark_url
       @transaction_id = params[:id]
-      @shop_coverage_result = params[:shop_coverage_result]
+      @shop_coverage_result ||= params[:shop_coverage_result]
```

Comment:
Should this be a boolean or string? look what is the value on review_and_submit page, this should be similar to that.